src/parsers/babel_parser.js

```diff
@@ -171,3 +170,20 @@ export const parse = (file: string, data: ?string): ParseResult => {
 
   return parseResult;
 };
+
+export const plugins = [
+  'jsx',
+  'classProperties',
+  'exportDefaultFrom',
+  'logicalAssignment',
+  'nullishCoalescingOperator',
+  'objectRestSpread',
+  'optionalChaining',
+  'topLevelAwait',
+];
+
+// Its not possible to use the parser with flow and typescript active at the same time
+export const parseJs = (file: string, data: ?string, additionalPlugins: string[] = []): ParseResult =>
+  parse(file, data, {plugins: [...plugins, ...additionalPlugins, 'flow']});
+export const parseTs = (file: string, data: ?string, additionalPlugins: string[] = []): ParseResult =>
+  parse(file, data, {plugins: [...plugins, ...additionalPlugins, 'typescript']});
```

Comment:
giving the `plugins` here are just "flags" for the babel-parser, I suggest we add all of them, you can reference their list [here](https://github.com/babel/babel/blob/928b9f8c9518284eac6d0598633f2ec373fc6d0c/packages/babel-parser/typings/babel-parser.d.ts#L97).

Once we enabled everything, there is no point to add "additional"... the customization should be very rare, if any, and it could be simply an override.

Reply:
it was not possible to add every plugin, but i added almost every plugin. i removed the ones which caused conflicts.

Reply:
i wonder if adding all those plugins is even necessary